apps/azure-functions-api/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "azure-functions-api",
+  "version": "0.0.1",
+  "description": "Azure Function REST API Template",
+  "main": "./dist/main.js",
+  "license": "MIT",
+  "scripts": {
+    "clean": "shx rm -rf dist",
+    "clean:generated": "shx rm -rf ./src/generated",
+    "build": "tsc",
+    "build:watch": "yarn build -w",
+    "typecheck": "yarn build --noemit",
+    "extensions:install": "func extensions install",
+    "start": "dotenv -e .env func start --javascript",
+    "start:watch": "yarn start watch",
+    "lint": "yarn lint:check --fix",
+    "lint:check": "eslint . -c .eslintrc.js --ext .ts,.tsx --cache",
+    "lint:api": "sh api-lint.sh",
+    "api:bundle": "bundle-api-spec -i api/internal.yaml.template -o api/internal.yaml -V $npm_package_version",
+    "generate": "turbo run generate:definitions-internal",
+    "generate:definitions-internal": "shx mkdir -p ./src/generated/definitions/internal && gen-api-models --api-spec ./api/internal.yaml --no-strict --out-dir ./src/generated/definitions/internal "
+  },
+  "devDependencies": {
+    "@pagopa/eslint-config-node": "*",
+    "@pagopa/typescript-config-node": "*",
+    "@types/express": "^4.17.12",
+    "@types/node": "~20.10.0",
+    "dependency-check": "^4.1.0",
+    "dotenv-cli": "^7.4.1",
+    "eslint": "^8.57.0",
+    "eslint-plugin-prettier": "^5.1.3",
+    "prettier": "^3.2.5",
+    "shx": "^0.3.2",
+    "swagger-cli": "^4.0.4",
+    "typescript": "^5.4.3"
+  },
+  "dependencies": {
+    "@azure/functions": "4.2.0",
+    "@pagopa/handler-kit": "^1.1.0",
+    "@pagopa/handler-kit-azure-func": "^2.0.1",
+    "@pagopa/io-functions-commons": "^28.21.0",
+    "@pagopa/openapi-codegen-ts": "^13.1.0",
+    "@pagopa/ts-commons": "^12.6.0",
+    "azure-storage": "^2.10.7",
```

Comment:
[azure-storage](https://www.npmjs.com/package/azure-storage) is deprecated

![image](https://github.com/pagopa/dx-typescript/assets/4357400/7692e4a2-5ca3-40a5-847f-0d6aa36f3e96)

Reply:
Removed in 4e3672d. But this lib is a dependency of `io-functions-commons`, should be replaced even on that lib for a complete upgrade.